apycula/pindef.py

```diff
@@ -109,19 +109,19 @@ def get_diff_cap_info(device, package, special_pins=False):
         is_diff = 'DIFF' in pin.keys()
         if not is_diff:
             res[str(pin['NAME'])] = (is_diff, is_true_lvds, is_positive)
-            cotinue
```

Comment:
This led me to believe that this situation never occurs in principle, otherwise it would have been discovered a long time ago, but left it for now just in case :)